src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-card-pop-up/ubs-admin-tariffs-card-pop-up.component.ts

```diff
@@ -27,7 +27,7 @@ export class UbsAdminTariffsCardPopUpComponent implements OnInit, OnDestroy {
     station: ['', Validators.required],
     regionNameUk: ['', Validators.required],
     regionNameEng: [''],
-    city: [{ value: '' }, [Validators.maxLength(40), Validators.required]]
+    city: [{ value: '', disabled: false }, [Validators.maxLength(40), Validators.required]]
```

Comment:
Change city: **disabled: true**
Add on ngOnInit() - region.valueChanges.pipe(takeUntil(this.unsubscribe)).subscribe( res => {
  if(res) {
  this.city.enable()
  }
}